js/src/elements/form-behavior.ts

```diff
@@ -2,6 +2,7 @@ import { generateElementId, kebabToCamelCase } from '../utilities/util';
 
 export const defaultMessageStrings = {
 	contentHasChanged: 'Content has changed, please reload the page to get the latest changes.',
+	inputGroupRequired: 'At least one selection for {inputGroup} is required.',
```

Comment:
Is this the correct validation string for radio?

<img width="380" alt="image" src="https://github.com/user-attachments/assets/9f5a4fd8-d442-438c-ab62-209b736e6044">

Reply:
That is the text of the `.field-label` for the radio.  So yes.

Reply:
Looks better now